src/platform/Tizen/ConnectivityManagerImpl.cpp

```diff
@@ -224,6 +253,55 @@ bool ConnectivityManagerImpl::IsWiFiManagementStarted()
     return isActivated;
 }
 
+CHIP_ERROR ConnectivityManagerImpl::GetWiFiBssId(ByteSpan & value)
+{
+    uint8_t * bssId = nullptr;
+    CHIP_ERROR err = Internal::WiFiMgr().GetBssId(bssId);
+    ReturnErrorOnFailure(err);
+
+    value = ByteSpan(bssId, 6);
```

Comment:
This should be updated to the API change that landed for GetWiFiBssid, where now caller passes in the buffer to place the BSSID into.

Reply:
I changed to `MutableByteSpan` from `ByteSpan`. but i will leave it as not implementation now.
And we need to suggest to change API signature as you suggested.

Reply:
The API signature already changed.